WordPress/WordPressShare/BlogPickerViewController.swift

```diff
@@ -0,0 +1,135 @@
+import Foundation
+import UIKit
+import WordPressShared
+import WordPressComKit
+
+
+class BlogPickerViewController : UITableViewController
```

Comment:
On top of that, there's already a `BlogSelectorViewController`. I understand we have two because the extension can't hit core data, but can we make the names reflect the differences between both?